### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by Claire1217**
Name: Wenjun Zhang
Student ID: s4469251
Description: This module contains a solution to ISICs data segmentation using improved unet. Dice similarity coefficient at 0.82. 
The directory contains a Script.ipynb file, inside which is the main function running the segmentation. Improved_unet.py contains the tensorflow model. The folder images contains all the plots and images used in README.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Code completed & submitted in time
- Code valid for Easy Difficulty
- Could use more detailed commit messages
- Results are good

Other Feedback:
- spread out your commits in the future if possible.

**Comment by rexsw** [comment]
Code looks good for normal difficulty

For next time;

Make sure you clean up any leftovers when you switch from .ipy to .py i.e #In[4]

Code in files like driver scripts should be in a main function and in a python file

File: recognition/s4469251/ImprovedUnet.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
```
Comment (shakes76) [inline recognition/s4469251/ImprovedUnet.py:2]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info. 
Notebooks have markdown sections for you to explain what's going on, so you should make use of it.

**Pull request merged**